packages/cmf/src/sagas/http.js

```diff
@@ -45,7 +50,8 @@ export function handleBody(response) {
  * @return {Promise}           A promise that reject with the result of parsing the body
  */
 export function handleError(response) {
-	return handleBody(response).then(body => new HTTPError(body));
+	// BREAKING CHANGE to allow Go-like error handling
```

Comment:
I think this BR should exists in the PR description to think to move into the wiki on release